CardScanSystemTest/CardScanSystemTest/Resources/.gitignore

```diff
@@ -0,0 +1 @@
+apikey.json
```

Comment:
are we handling api keys this way only for systems test?

Reply:
yes, only for system test